packages/features/bookings/Booker/Booker.tsx

```diff
@@ -132,6 +133,11 @@ const BookerComponent = ({
   };
 
   const shouldShowFormInDialog = shouldShowFormInDialogMap[layout];
+
+  if (bookerState === "loading") {
+    return <Spinner />;
```

Comment:
I don't think we want to show a spinner. I would prefer to return `null` if `bookerState` is still loading. 

Let me know in case I am missing something and it was mentioned somewhere to use a spinner here (cc @Jaibles )

Reply:
updated


https://github.com/calcom/cal.com/assets/32460534/9931f492-bedb-4df6-8ff3-6cfdaf927d26